### ISSUE TITLE
Future QoL Improvements: Graduation / Scale

### ISSUE THREAD

**Issue by jufcar** [loki79uk/FS25_UniversalAutoload#6]
I've no idea of lua, just read in anothert WIP Mod that with FS25 UI changed a lot, so that my idea cannot be implemented for the time being.
But it would be nice to get some kind of measurement when editing faces if i try to reach a certain lenght/width.

**Comment by loki79uk** [comment]
Yes agreed - that is already on my todo list.  I could even give feedback of multiples of common bale sizes, etc..  but it is hard to do that for every pallet.

**Comment by loki79uk** [comment]
I'm closing this now as the "temporary" solution of displaying the dimensions at the top of the screen seems to be as good as I can come up with.  I might still add an option to round or manually enter specific values.
